ncm-cron/src/main/pan/components/cron/schema.pan

```diff
@@ -27,40 +27,123 @@ type structure_cron_syslog = {
     Define specific attributes for cron log file.
 }
 type structure_cron_log = {
-    @{A boolean disabling the redirection of script output/error to a log file}
+    @documentation{A boolean disabling the redirection of script output/error to a log file}
```

Comment:
please don't change this. we don't do this for any attribute, only for the actual types. i'm nit sure the docs are generated correctly when this is changed